pkg/controller/nativeResourceWorker.go

```diff
@@ -678,7 +678,7 @@ func (ctlr *Controller) processConfigMap(cm *v1.ConfigMap, isDelete bool) (error
 	} else if len(es.ExtendedRouteGroupConfigs) > 0 {
 		ergc := es.ExtendedRouteGroupConfigs[0]
 		if ergc.Namespace != cm.Namespace {
-			return fmt.Errorf("Invalid Extended Route Spec Block in configmap: %v/%v", cm.Namespace, cm.Name), true
+			return fmt.Errorf("Invalid Extended Route Spec Block in configmap: Multiple namespaces found in %v/%v", cm.Namespace, cm.Name), true
```

Comment:
Vidhi, can we log to mention the correct namespace at index 0 - as users are free to use multiple namespaces